src/Models/ProductModel.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+require_once 'src/Entities/Product.php';
+
+class ProductModel
+{
+    public static function getProductList($cat_id, PDO $db): array
```

Comment:
missing type hint on `$cat_id`

Reply:
also this isnt a great method name. It gets products by category ID, lets be more specific